src/NServiceBus.Core/Configure.cs

```diff
@@ -126,6 +127,21 @@ internal void Initialize()
             }
         }
 
+        void ConfigureBehaviors()
+        {
+            var pipeLineExecutor = Builder.Build<PipelineExecutor>();
+            var incomingBehaviorTypes = pipeLineExecutor.Incoming.Select(registerStep => registerStep.BehaviorType).ToArray();
+            var outgoingBehaviorTypes = pipeLineExecutor.Outgoing.Select(registerStep => registerStep.BehaviorType).ToArray();
+            foreach (var outgoingBehaviorType in outgoingBehaviorTypes)
+            {
+                configurer.ConfigureComponent(outgoingBehaviorType, DependencyLifecycle.InstancePerCall);
```

Comment:
@timbussmann I do know that some features do some property configuration on behaviors.

I just checked the `IConfigureComponents` documentation and that isn't clear about if last or first writer wins for a type. All AT's do run correctly so maybe that is due to Autofac that has a first writer wins policy?

Based on that It is probably wise to check if there is already a registration for that Type.